src/System.Collections.NonGeneric/tests/SortedListTests.cs

```diff
@@ -1003,17 +989,17 @@ public static void TestGetValueList_CopyTo(int count, int index)
         }
 
         [Fact]
-        public static void TestGetValueList_CopyTo_Invalid()
+        public static void GetValueList_CopyTo_Invalid()
         {
             SortedList sortList1 = Helpers.CreateIntSortedList(100);
             Helpers.PerformActionOnAllSortedListWrappers(sortList1, sortList2 =>
             {
                 IList values = sortList2.GetValueList();
-                Assert.Throws<ArgumentNullException>(() => values.CopyTo(null, 0)); // Array is null
-                Assert.Throws<ArgumentException>(() => values.CopyTo(new object[10, 10], 0)); // Array is multidimensional
+                Assert.Throws<ArgumentNullException>("dest", () => values.CopyTo(null, 0)); // Array is null
+                Assert.Throws<ArgumentException>("array", () => values.CopyTo(new object[10, 10], 0)); // Array is multidimensional
 
-                Assert.Throws<ArgumentOutOfRangeException>(() => values.CopyTo(new object[100], -1)); // Index < 0
-                Assert.Throws<ArgumentException>(() => values.CopyTo(new object[150], 51)); // Index + list.Count > array.Count
+                Assert.Throws<ArgumentOutOfRangeException>("dstIndex", () => values.CopyTo(new object[100], -1)); // Index < 0
```

Comment:
Unfortunately, I don't think I have control of the parameter name, as the implementation of CopyTo across the lib doesn't perform any argument validation. 

For example, the Param name here is "index" but the thrown Param is "dstIndex"

I considered adding custom argument validation in these situations, but I'm not certain about the Perf effect or changing legacy code